MatrixSDK/Crypto/Verification/MXDeviceVericationManager.m

```diff
@@ -45,6 +45,8 @@
 // Request timeout in seconds
 NSTimeInterval const MXRequestDefaultTimeout = 5 * 60.0;
 
+static NSArray<MXEventTypeString> *MXDeviceVerificationManagerDMEventTypes;
```

Comment:
Prefer use k prefix for local constants like `kMXDeviceVerificationManagerDMEventTypes`